ui/raidboss/data/03-hw/trial/sophia-ex.js

```diff
@@ -406,27 +437,33 @@
       netRegexCn: NetRegexes.startsUsing({ id: '19A9', source: '索菲娅' }),
       netRegexKo: NetRegexes.startsUsing({ id: '19A9', source: '소피아' }),
       condition: function(data) {
-        // No platform tilts if clones are up, and no tilts if it's the first one.
-        return !data.clonesActive && data.scaleSophias && data.sadTethers;
+        return data.sadTethers && data.scaleSophias;
```

Comment:
Hmm. You're right, we can't put `firstTethers` or similar anywhere since we don't know which set of tethers will *be* first. What if instead we just completely ignored whether or not the first set of tethers appeared, apart from a check for "is this set balanced?" within the cast call trigger? We already have all other balanced tether sets accounted for via `clonesActive`, so I think it should work fine.

Reply:
Yeah, I think "don't call out when balanced" will cover everything without a special case.  Probably worth a comment about this logic too, since it won't be apparent when reading.